### PR TITLE
Self service UX tweaks

### PR DESCRIPTION
**Pull request by neoeno**
Collection of little frontend-y bits in here. Commits should explain, feel free to ask for further explanation if anything's unclear.

### CODE REVIEW

File: app/assets/stylesheets/layouts/_half_width.scss [hunk 1]
```diff
@@ -1,4 +1,8 @@
 .l-half-width {
-  padding-right: $gutter-width/2;
-  width: percentage(1/2);
+  width: 100%;
+
+  @include respond-to($mq-l) {
+    padding-right: $gutter-width/2;
+    width: 50%;
```
Comment (neoeno) [inline app/assets/stylesheets/layouts/_half_width.scss]: Was feeling so Monday earlier that I thought the `percentage` function was the dough columns function.

Anyway, it turned out that using `@include column(6);` here would involve either doing some float clearing or adding a row element, so doing it with straight percentages seemed more straightforward.

**Comment by urfolomeus** [comment]
LGTM :+1:

**Comment by benbarnett** [comment]
:+1:

**Comment by neoeno** [comment]
Thanks!

**Pull request merged**